### DISCUSSION TITLE
Relative Beatmap Leaderboard (League Ranks?)

### DISCUSSION THREAD

**Discussion by KaapaTeelb** [ppy/osu#25192] [Ideas]
Idea: Only display 10-ish% of players around your rank in map leaderboards.
Aka, ~5% below, ~5% above.

Reasoning: I recently switched to lazer and noticed not too much players in there. A bunch of maps had people around my rank and not those super buffed top 100 dudes that make any map score impossible to compete with.
And I thought it was kind of cool.
If there would be some sort of league ranks, like 1-1000, 1001-3000, 3001-7000, 7001-15000, 15001-35000, 35001-70000, 70001-150000 etc?? (Not too good at balance in those things) That had their separate leaderboards, Would (probably) help players fight for their spots in leagues idk :D

(Couldn't find any post like this)

**Comment by peppy** [comment]
this is the eventual plan!
